plugins/CorePluginsAdmin/Controller.php

```diff
@@ -316,7 +316,9 @@ public function safemode($lastError = array())
 
             $errorMessage = $lastError['message'];
 
-            if (!empty($lastError['backtrace'])) {
+            if (!empty($lastError['backtrace'])
```

Comment:
@diosmosis is it on purpose that this is only done when output format is not HTML? I wonder if it is supposed to be shown on the safemode page when backtrace with message is enabled? I think before it was the case

Reply:
AFAIK there was no backtrace to display in the code before (but I never paid that much attention to the safemode html).

Reply:
I can display it in the View as well.